pages/about.js

```diff
@@ -8,18 +8,18 @@ export default function About() {
   return (
     <Layout>
       <section className={`${utilStyles.wrapper__content} ${utilStyles.wrapper__about}`}>
-        <div className={utilStyles.caption} data-aos="fade-up" data-aos-delay="100" data-aos-easing="ease-in-out-quad">
           <h1 className={utilStyles.title}>Health</h1>
           <h2 className={`${utilStyles.title} ${utilStyles.titleStrong}`}>Connect</h2>
           <h3 className={utilStyles.subtitle}>Our responses to COVID-19</h3>
           <p className={utilStyles.description}>
-            With countries experiencing a massive need for direct communication about COVID-19, {' '}
-            <a href="https://www.praekelt.org/">Praekelt.org</a> (using <a href="https://www.turn.io/">Turn.io</a>)
-            has developed a pro bono solution to empower citizens with the right
-            information and support the national
-            response of the health systems that serve them.
+            With countries experiencing a massive need for direct communication
+            about COVID-19, Praekelt.org (using Turn.io) has developed a pro bono
+            solution to empower citizens with the right information and support
+            the national response of the health systems that serve them.
```

Comment:
how come we are removing the links?

Reply:
Thanks for pointing that out. They aren't in the copy but I'll check if they are supposed to be there

Reply:
Confirmed that these links shouldn't be here.